docs/LibraryEvolution.rst

```diff
@@ -145,14 +145,22 @@ The following changes are permitted:
   body, not the labels that are part of the function's full name).
 - Reordering generic requirements (but not the generic parameters themselves).
 - Adding a default argument expression to a parameter.
-- Changing or removing a default argument is a `binary-compatible
-  source-breaking change`.
+- Adding, changing, reordering, or removing property wrappers that either are 
+  implementation-detail or in a composition where the outermost wrapper is
+  implementation-detail (``@ImplementationDetail @MyWrapper``).
+- Changing or removing a default argument is a `binary-compatible source-breaking change`.
 - Adding or removing the ``@discardableResult`` and ``@warn_unqualified_access``
   attributes.
 
 No other changes are permitted; the following are particularly of note:
 
 - An ABI-public function may not change its parameters or return type.
+- An ABI-public function may not, in any way, change API-level property 
+  wrappers (``@API``) or compositions where the outermost wrapper is 
+  API-level (``@API @MyWrapper``).
```

Comment:
Nit: these still read as though there's some built-in feature named `@API` (or `@ImplementationDetail`); if I understand @hborla's point, I think all of these examples in parens can be deleted.

Reply:
OK, I'll remove them. Are the bullet points clear without any examples, though, or would you like to see a different kind of example?